docs/source/technical_tutorials/access/direct_access.ipynb

```diff
@@ -0,0 +1,1564 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #24.        return fsspec.filesystem(
</pre><p>We probably need a Rasterio example somewhere too.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/344/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>